### PR TITLE
Create 2019-06-06-why-should-we-be-present-at-events.md

### PR DESCRIPTION
**Pull request by github-luckow-org**
new blog post. please review

### CODE REVIEW

File: blog/2019-06-06-why-should-we-be-present-at-events.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+A second highlight of the Hackathon event was a 20 minute talk to a Bosch guy. He introduced himself as **the biggest fan of our project**. 
+I‘ve never heard from him. He told me that he has no time to be part of our community. But he really likes the strong and sharing spirit of our community. Later on he gave me his business card and told me if we ever need help to address the right people in the Bosch universe he wants to help me finding the right person. Check! [target groups 2 and 3]
+
+Another vibrant event immediately followed the really cool Bosch Hackathon - I attended the 
+
+## Maker Faire Berlin. 
+Attendees: the do-it-yourself (DYI) community. Starting on Friday with a day exclusively for school children. That is one of our target groups. The young talents. We need them [target group 1] because many of us are "really old" and as a sustainable open source software community we have to hand the project over to the younger ones. You can imagine the fun with young people if you show them the smart home. They spontaneously understand the dependencies.
```
Comment (davidgraeff) [inline blog/2019-06-06-why-should-we-be-present-at-events.md]: Start with a capital after the colon?

File: blog/2019-06-06-why-should-we-be-present-at-events.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+On Sunday I gave a session in the venue cinema with the topic “What is the 'smart' in smart home?”. [all target groups]
+
+After 7 days in a row of supporting our fantastic project openHAB, I felt a bit like never ever talking about home automation again :) 
+One week later I attend the Smart Home Day in Darmstadt ... But that is another story.
+
+My call to action for our community: Keep in mind (at every occasion) that we have marketing goals. Let's turn interested people into amateurs and turn amateurs into community members. Let's grow the community! 
+
+I want to thank the people who helped making these events happen. Take a look at their faces on twitter. 
+https://twitter.com/luckow/status/1129754848474423298<br />
```
Comment (davidgraeff) [inline blog/2019-06-06-why-should-we-be-present-at-events.md]: Turn that into a real list maybe? A small twitter emblem before each link would be sweet as.

File: blog/2019-06-06-why-should-we-be-present-at-events.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+layout: BlogPost
+category: blog
+title: Why should we – as the openHAB community – be present at events?
+author: Stephan Luckow
+date: '2019-06-06'
+previewimage: /uploads/190606_why_be_present.jpg
+tags:
+  - news
+excerpt: >-
+  My call to action for our community: Keep in mind (at every occasion) that we have marketing goals. Let's turn interested people into amateurs and turn amateurs into community members. Let's grow the community!
```
Comment (ghys) [inline blog/2019-06-06-why-should-we-be-present-at-events.md]: @github-luckow-org could you add double-quotes (") around this part, or rephrase to remove the colon
```
  "My call to action for our community: Keep in mind (at every occasion) that we have marketing goals. Let's turn interested people into amateurs and turn amateurs into community members. Let's grow the community!"
```
That's what makes the build choke.

File: blog/2019-06-06-why-should-we-be-present-at-events.md (hunk 1 above)
Comment (github-luckow-org) [reply blog/2019-06-06-why-should-we-be-present-at-events.md]: yep. check

File: blog/2019-06-06-why-should-we-be-present-at-events.md (hunk 2 above)
Comment (github-luckow-org) [reply blog/2019-06-06-why-should-we-be-present-at-events.md]: check. (sorry without twitter emblem)

File: blog/2019-06-06-why-should-we-be-present-at-events.md (hunk 3 above)
Comment (github-luckow-org) [reply blog/2019-06-06-why-should-we-be-present-at-events.md]: double-quotes check

**Comment by github-luckow-org** [comment]
hah. never before i pushed the "close and comment" button. this is my first time. thanks for your time for reviewing.

**Comment by github-luckow-org** [comment]
oh. i think i better reopen :)

**Comment by kaikreuzer** [comment]
Very nice, many thanks @github-luckow-org!
Just one cosmetic question: Woud it be possible to replace the (rather ugly) url list at the end by clickable images or at least by the ambassador names behind the links? I am not sure whether it is possible to put images next to each other horizontally, though (@ghys, do you happen to know?).

**Comment by github-luckow-org** [comment]
tried my best to solve both wishes (pics to the post and little twitter icons). the result is far away from perfect. someone with (web)design skills?

**Comment by github-luckow-org** [comment]
i save life time. no ugly list anymore. now there is only one link to a twitter "moments".

**Comment by ghys** [comment]
😄 
I was just writing:
Probably makes sense to allow embedding tweets (limited to blog posts). I think it's only a matter of declaring the component on the blog post page template - the library is there for the home page anyway.
I can have a look eventually.

**Pull request merged**